### ISSUE TITLE
Jaws reads out long decimals

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/wave-on-a-string#205]
**Device**
Samsung laptop
**OS**
Win 11
**Browser**
Chrome + Jaws
**Problem Description**
For https://github.com/phetsims/qa/issues/1289. Similar to known issue https://github.com/phetsims/resistance-in-a-wire/issues/177.
When Jaws is reading out the values on the sliders, not just max/min values, it will often read out long and repeating decimals.
**Visuals**

https://github.com/user-attachments/assets/b27c3278-251b-41f7-97b4-82221c1cedfa

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe I will take a look.

**Comment by KatieWoe** [comment]
Do you mean the alerts? They seem to be playing for me?

**Comment by jessegreenberg** [comment]
The value that the screen reader should be reading is this in the a11y view:

<img width="488" height="58" alt="Image" src="https://github.com/user-attachments/assets/b1fb0fd8-9826-4c58-849b-adce1ad1c1d9" />

But instead, it just reads the number with many decimals as you noted in the issue. I don't hear "centimeters" at all.

**Comment by KatieWoe** [comment]
Ah. Would you like a separate issue for that? Sorry I didn't catch it earlier

**Comment by jessegreenberg** [comment]
No, thats OK thanks - I think it is all related to this one.

**Comment by jessegreenberg** [comment]
This appears to be a recent regression with JAWS, see https://github.com/FreedomScientific/standards-support/issues/916.

**Comment by jessegreenberg** [comment]
OK, I commented in that issue, submitted a ticket directly to Freedom Scientific, and added this to our list of known accessibility bugs.

@KatieWoe is there anything else you think we should do or can this be closed?

**Comment by KatieWoe** [comment]
I can't think of anything. Closing
